CHANGELOG.md

```diff
@@ -6,30 +6,22 @@
 
 - Assertions now print their callstack a log file ([#637](https://github.com/getsentry/sentry-unreal/pull/637))
 - Add UE 4.27, 5.0 and 5.1 to CI pipeline ([#675](https://github.com/getsentry/sentry-unreal/pull/675))
+- Download the `Cocoa SDK` instead of building it ([#681](https://github.com/getsentry/sentry-unreal/pull/681))
```

Comment:
I don't think this needs to be in the changelog since for the end user, it doesn't make a difference, does it?

Reply:
You're right, this can be removed.